src/components/02-elements/cookie-consent/cookie-consent.njk

```diff
@@ -0,0 +1,37 @@
+<head>
+    <!-- Matomo -->
+    <script type="text/javascript">
+      var _paq = window._paq = window._paq || [];
+      /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+      _paq.push(['requireCookieConsent']); // Don't use cookies unless we have consent
+      _paq.push(['setCookieDomain', '*.threesixtygiving.org']);
+      _paq.push(['setDomains', '*.threesixtygiving.org']);
+      _paq.push(['setDownloadExtensions', "json|csv|xlsx"]);
+      _paq.push(['trackPageView']);
+      _paq.push(['enableLinkTracking']);
+      (function() {
+        var u="https://analytics.threesixtygiving.org/";
+        _paq.push(['setTrackerUrl', u+'matomo.php']);
+        _paq.push(['setSiteId', '26']);
```

Comment:
If we leave `setSiteId` `26` here a visit may be registered for Quality Dashboard which would not be ideal. This could be a global variable or some other way to make sure the component is customised to the context.